test/passenger_test.rb

```diff
@@ -69,6 +80,191 @@
   end
 
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+
+    it "sums up one trip's cost" do
+      @trip = RideShare::Trip.new(
```

Comment:
The use of `@` can be a little confusing in tests. It's only needed in a `before` block when you have a variable you want to use in multiple tests.